### DISCUSSION TITLE
Wagtail Admin: autocomplete_fields

### DISCUSSION THREAD

**Discussion by sowinski** [wagtail/wagtail#11980] [Q&A] [answered]
Hi,

is it possible to use autocomplete fields somehow in the wagtail admin?
I have list of cities and this is to much to handle for a normal dropdown menu. 
I guess a lot of people have problems with to much elements and I am wondering if I can use the "autocomplete_fields" from django admin somehow also in the wagtail admin?

Thank you

**Comment by gasman** [comment, marked as answer]
Have you seen https://wagtail-autocomplete.readthedocs.io/ ?

**Reply by sowinski** [reply]
Thank you for the fast reply
